src/Altinn.App.Core/Features/IProcessExclusiveGateway.cs

```diff
@@ -18,6 +18,7 @@ public interface IProcessExclusiveGateway
     /// </summary>
     /// <param name="outgoingFlows">Complete list of defined flows out of gateway</param>
     /// <param name="instance">Instance where process is about to move next</param>
+    /// <param name="action">Action performed by the requester</param>
     /// <returns></returns>
-    public Task<List<SequenceFlow>> FilterAsync(List<SequenceFlow> outgoingFlows, Instance instance);
+    public Task<List<SequenceFlow>> FilterAsync(List<SequenceFlow> outgoingFlows, Instance instance, string? action);
```

Comment:
Is it correct that an exclusive gateway returns a list of sequence flows?

Reply:
Had the same thought. We need to decide if it should be possible to return more than one flow from a gateway and then choose the default (if defined), or if a gateway always needs to return one flow